src/components/Footer/Footer.css

```diff
@@ -0,0 +1,7 @@
+.footer {
+    display: flex;
+    justify-content: left;
+    align-items: center;
+    background-color: #333;
+    color: #fff;
+}
```

Comment:
Esta forma de escribir estilos tiene una desventaja: si en otro archivo se escribe una clase con el mismo nombre se genera un conflicto y se puede tener un comportamiento no deseado en el app

[Aquí](https://medium.com/web-for-you/css-conflict-resolution-rules-explained-9ab90de18aef) hay un artículo con mas información al respecto

Una forma de solucionar esto es usando [CSS Modules](https://create-react-app.dev/docs/adding-a-css-modules-stylesheet/)